### PR TITLE
OpenAPI Update for 17.7

### PR DESCRIPTION
**Pull request by ted-spence-avalara**
Patch notes: https://developer.avalara.com/blog/2017/07/25/AvaTax-17-7-patch-notes

### CODE REVIEW

File: src/AvaTaxClient.php [hunk 1]
```diff
@@ -6604,488 +7146,507 @@ class BatchFileModel
...
     /**
-     * @var string If this company participates in Streamlined Sales Tax, this is the date when the company joined the SST program.
+     * @var int TODO
      */
-    public $sstEffectiveDate;
+    public $taxNumber;
 
     /**
-     * @var string The two character ISO-3166 country code of the default country for this company.
+     * @var boolean TODO
```
Comment (nitinshirsat) [inline src/AvaTaxClient.php]: Not a blocker but we need to remove TODOs in comments

File: src/AvaTaxClient.php [hunk 2]
```diff
@@ -562,6 +595,24 @@ public function getCompany($id, $include)
...
 
     /**
+     * 
+     *
+     * 
+     *
+     * 
+     * @return FetchResult
+     */
+    public function getCompany()
```
Comment (nitinshirsat) [inline src/AvaTaxClient.php]: getCompany is already declared in class. Method polymorphism is not available in PHP.

Comment (ted-spence-avalara) [inline src/AvaTaxClient.php]: Roger that.  We found an error in the original swagger file; I'll correct it.

File: src/AvaTaxClient.php (hunk 1 above)
Comment (ted-spence-avalara) [inline src/AvaTaxClient.php]: We found quite a few in 17.7; they are being corrected in the 17.8 release, and we're adding an integration test that ensures that the word "TODO" never appears in the swagger file.

**Comment by ted-spence-avalara** [comment]
Updated with feedback from Nitin; broke single-file API apart into separate files for readability

**Pull request merged**